cmd/move_config.go

```diff
@@ -66,12 +68,18 @@ func Move(dryRun bool) error {
 
 	log.Println("WARNING: Helm v3 configuration may be overwritten during this operation.")
 	log.Println()
-	doCleanup, err := utils.AskConfirmation("Move Config", "move the v2 configuration")
-	if err != nil {
-		return err
+	if skipConfirmation {
+
+		log.Println("Skipping confirmation before performing move.")
```

Comment:
move --> move config

Reply:
This has not been changed.

Reply:
Can you change: "Skipping confirmation before performing move." to "Skipping confirmation before performing move configuration."